### ISSUE TITLE
midi playback behaves unexpectedly with b5 and #5 chords

### ISSUE THREAD

**Issue by chrisfargen** [paulrosen/abcjs#365]
This project is amazing. Thank you for your hard work.

Chord symbols with flat-5 or sharp-5 are being interpreted with a regular 5 on boom2.

Screenshot below, with my current workaround.

![thing](https://user-images.githubusercontent.com/2993720/78945506-767bf000-7a8e-11ea-8fcf-dbe6d7550fbe.jpg)

```X: 1
T: Example #1
M: 4/4
L: 1/4
R: 
K: Amin
Q: 100
%
%%text Playback sounds weird:
%
"Am" E2 D C | "G#°7" D2 "_(↑); expected: G♯°7/E; actual: G♯°7/D♯" E D |
%
%%text Workaround:
%
"Am" E2 D C y| "G#°7" D2 "G♯°7/E" "_(↑)" E D |

**Comment by paulrosen** [comment]
Thanks for the report. It will be fixed in the next version.

**Comment by paulrosen** [comment]
Will be fixed in 6.0.0-beta.10
